expo-example/webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path')
```

Comment:
webpack is deprecated. We should use metro:

https://docs.expo.dev/router/migrate/from-expo-webpack/